src/aleph/http.clj

```diff
@@ -52,8 +52,7 @@
    | `idle-timeout` | when set, connections are closed after not having performed any I/O operations for the given duration, in milliseconds. Defaults to `0` (infinite idle time).
    | `continue-handler` | optional handler which is invoked when header sends \"Except: 100-continue\" header to test whether the request should be accepted or rejected. Handler should return `true`, `false`, ring responseo to be used as a reject response or deferred that yields one of those.
    | `continue-executor` | optional `java.util.concurrent.Executor` which is used to handle requests passed to :continue-handler.  To avoid this indirection you may specify `:none`, but in this case extreme care must be taken to avoid blocking operations on the handler's thread.
-   | `shutdown-quiet-period` | optional period in seconds for which new connections will still be serviced after a scheduled shutdown via `java.io.Closeable#close`. Defaults to 2 seconds.
-   | `shutdown-timeout` | optional grace period in seconds on which to wait for the event loop group to empty during a scheduled shutdown. Defaults to 15 seconds."
+   | `shutdown-timeout` | interval in seconds within which in-flight requests must be processed, defaults to 15 seconds. A value of 0 bypasses waiting entirely."
```

Comment:
Maybe this got lost in the conversation, but we shouldn't just delete an existing option. Can we make `shutdown-quiet-period` a subordinate alias for `shutdown-timeout`. (Subordinate in the sense that `shutdown-timeout wins if both are present.)

We can then either deprecate the option in the docstring, or remove it from the docstring entirely.

Reply:
It wasn't lost, I'm on @arnaudgeiser's side: we added this option two commits ago (9 days ago) and it never made it into a release, so there are no consumers out there

Reply:
For the record : https://github.com/clj-commons/aleph/pull/642#discussion_r1018233370

Reply:
Ah, OK, I missed Arnaud's comment about it. nm